app/assets/css/pages/_documentation.styl

```diff
@@ -336,3 +336,11 @@ hr.clear
                 // ideally this would be a slide out menu using Twitter Bootstrap Collapse
                 // like simple_fb_html5_sidebar
                 display: none
+
+@media only screen and (max-width: 480px)
+	.documentation
+        #content
```

Comment:
This indentation is confusing. It looks like you're using a combination of tabs and spaces. AFAIK we only use spaces here.